rafs/src/metadata/layout/v6.rs

```diff
@@ -1413,11 +1423,23 @@ struct RafsV6Blob {
     // SHA256 digest of RAFS blob for ZRAN, containing `blob.meta`, `blob.digest` `blob.toc` and
     // optionally 'image.boot`. It's all zero for ZRAN blobs with inlined-meta, so need special
     // handling.
+    // When using encryption mod, it's reused for saving encryption key.
     blob_meta_digest: [u8; 32],
     // Size of RAFS blob for ZRAN. It's zero ZRAN blobs with inlined-meta.
+    // When using encryption mod, it's reused for saving encryption iv first 8 bytes.
     blob_meta_size: u64,
-
-    reserved2: [u8; 48],
+    // When using encryption mod, used for cipher_iv last 8 bytes.
+    // 0                  7                 15
+    // +------------------+------------------+
+    // |  blob_meta_size  | cipher_iv[8..16] |
+    // |     8bytes       |      8bytes      |
+    // +------------------+------------------+
+    //  \_         cipher_iv[0..16]        _/
+    cipher_iv: [u8; 8],
```

Comment:
ok, google tells that it is an cipher's `initialization vector`. But is it needed for decryption at all? If not, we should not save it in v6 blob.

Reply:
ok, after looking at the rest of the PR, I see that IV is needed for decryption as well.